packages/components/src/components/post-button/post-button.tsx

```diff
@@ -0,0 +1,243 @@
+import { Element, Component, Host, h, State, Prop, Watch } from '@stencil/core';
+import { version } from '@root/package.json';
+
+/**
+ * @slot default - Content to place in the `default` slot.<p>Markup accepted: <a href="https://developer.mozilla.org/en-US/docs/Glossary/Inline-level_content">inline content</a>.</p>
+ */
+@Component({
+  tag: 'post-button',
+  styleUrl: 'post-button.scss',
+  shadow: true,
+})
+export class PostButton {
+  @Element() host: HTMLPostButtonElement;
+
+  @State() classes: string;
+
+  /**
+   * Defines the class of the component.
+   */
+  @Prop() readonly class: string | null = null;
+
+  /**
+   * Defines the variant of the component.
+   */
+  @Prop() readonly variant: string = 'secondary';
+
+  /**
+   * Defines the size of the component.
+   */
+  @Prop() readonly size: null | 'sm' | 'rg' | 'md' | 'lg' = 'md';
```

Comment:
The new button uses words instead of abbreviations for naming the sizes (https://www.figma.com/design/JIT5AdGYqv6bDRpfBPV8XR/Foundations-%26-Components-Next-Level?node-id=1913-7309&t=OfKC9fyWzpVtdkZF-4). We should implement this here as well:
- Small
- Medium
- Large
- Extra large (extra-large)